packages/browser-client/src/Components/DisplayBlock.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Container,
+  Box,
+  TabPanels,
+  TabPanel,
+  Tab,
+  TabList,
+  Tabs,
+  Table,
+  Thead,
+  Tr,
+  Th,
+  Td,
+  Text,
+  Link,
+} from '@chakra-ui/react'
+// eslint-disable-next-line implicit-dependencies/no-implicit
```

Comment:
I don't know why I'm getting this error --
It compiles fine if I disable it like this